soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -54,8 +54,12 @@ const clock_enet_pll_config_t ethPllConfig = {
 	.enableClkOutput500M = true,
 #endif
 #ifdef CONFIG_ETH_MCUX
+#if defined(CONFIG_BOARD_MIMXRT1062_FMURT6)
```

Comment:
I am unclear why this `#if` is needed. It looks like the ETH clock output will be enabled if `CONFIG_ETH_MCUX` is set

Reply:
Ok, this makes sense. I would rather see us setting this up based on DT nodelabels then. Something like the following:

```
#if DT_NODE_HAS_STATUS(DT_NODELABEL(enet), okay)
enableClkOutput = true,
#endif
#if DT_NODE_HAS_STATUS(DT_NODELABEL(enet2), okay)
enableClkOutput1 = true,
#endif
```

Reply:
This is a valuable comment for me since this clearly marks the difference between Zephyr and Linux, as I think Linux doesn't have DT_HAS type unmodifiable Kconfig options which are based on Device tree.

Reply:
Made the changes as suggested

Reply:
@danieldegrasse Made the changes as suggested